packages/libs/tool-sdk/src/lib/constants.ts

```diff
@@ -0,0 +1 @@
+export const VINCENT_TOOL_API_VERSION = '2.0.0' as const;
```

Comment:
Shouldn't we get this from its package.json? We are going to forget updating this on release or version bump

Reply:
might be good to put this into the docs somewhere.  like make it clear what the different versions are in the diff packages and how they interact.  like "major version of the tool-sdk is compatible with app-sdk" etc, all versions right now are compatible with VINCENT_TOOL_API_VERSION 2.0.0, etc.